submissions/damilolaSalami/html-form/signUp.html

```diff
@@ -0,0 +1,91 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="signUp.css">
+    <title>Buy Me</title>
+</head>
+<body>
+    <nav>
+        <div class="header">
+            <div class="logo">
+                <img src="images/logo.jpg" alt="logo">
+            </div>
+           
+            <div class="headerlinks">
+                <ul>
+                    <li><a href="#">Product page</a></li>
+                    <li><a href="#">Contact Us</a></li>
+                    <li><a href="siguppage.html">Sign up</a></li>
+                </ul>
+            </div>
+        </div>
+    </nav>
+    
+<div class="mainbody">
+    <div class="registration container">
+        <h1>Register your<br>Account</h1> <br>
+        <p>Registering an account gives you <br>
+           access to sweet promo and events <br>
+           available to only registered <br>
+           customers.</p>
+        <br>
+        <p>Already got an account? <a href="#">Sign in</a></p> 
+    </div>
+
+    <section>
+        <form class="regform container">
+            <label for="firstname">First Name</label><br>
+            <input type="text" name="firstname" id="firstname" class="inputbox"><br>
+            <label for="lastname">Last Name</label><br>
+            <input type="text" name="lastname" id="lastname" class="inputbox"><br>
+            <label for="othernames">Other Names</label><br>
+            <input type="text" name="othernames" id="othernames" class="inputbox"><br>
+            <label for="phonenumber">Phone Number</label><br>
+            <input type="tel" name="phonenumber" id="phonenumber" class="inputbox"><br>
+            <label for="email">Email</label><br>
+            <input type="email" name="email" id="email" class="inputbox"><br>
+            <label for="password">Password</label><br>
+            <input type="password" name="password" id="password" class="inputbox"><br>
+            <label for="confirmpassword">Confirm Password</label><br>
+            <input type="password" name="confirmpassword" id="confrimpassword" class="inputbox"><br> <br>
+            <input type="checkbox" name="terms" id="terms" class="box">
+            <label for="terms"><p> I agree to the Terms and Conditions</p></label><br> <br>
+            <input type="button" value="SIGN UP" class="signup"> 
+        </form>
+    </section>
+</div>
+
+<footer class="footer">
+    <div class="mainfooter">
+        <div class="one containertwo">
```

Comment:
Using one, two or three as class names is not straightforward and explanatory to what it stands for. You need to update the names to be related to where is been used or what they are doing @queensaldam